mentat/llm_api.py

```diff
@@ -160,7 +160,9 @@ def is_model_available(model: str) -> bool:
 
 
 def model_context_size(model: str) -> Optional[int]:
-    if "gpt-4" in model:
+    if model == "gpt-4-1106-preview":
+        return 131072
```

Comment:
Where do you see this context size? I see [128000](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo) but maybe they announced a more precise number elsewhere?

Reply:
I was applying their previous definition of "K" but as they have documented 128K as 128,000 I will change this.